src/Core/src/Converters/EasingTypeConverter.cs

```diff
@@ -5,21 +5,18 @@
 using System.Reflection;
 using static Microsoft.Maui.Easing;
 
-namespace Microsoft.Maui.Controls
+#nullable disable
+
+namespace Microsoft.Maui.Converters
 {
-	/// <include file="../../docs/Microsoft.Maui.Controls/EasingTypeConverter.xml" path="Type[@FullName='Microsoft.Maui.Controls.EasingTypeConverter']/Docs" />
-	[Xaml.ProvideCompiled("Microsoft.Maui.Controls.XamlC.EasingTypeConverter")]
 	public class EasingTypeConverter : TypeConverter
 	{
-		/// <include file="../../docs/Microsoft.Maui.Controls/EasingTypeConverter.xml" path="//Member[@MemberName='CanConvertFrom']/Docs" />
```

Comment:
DO we want to remove these or should we just update this to the right place @hartez  ?